kernel/objects/capability-utils/objects/ops.hh

```diff
@@ -64,7 +64,7 @@ namespace mythos {
     optional<void> setReference(CapEntry& dst, Cap dstCap, CapEntry& src, Cap srcCap, const COMMITFUN& fun)
     {
       ASSERT(dstCap.isReference());
-      if (!dst.acquire()) THROW(Error::LOST_RACE);
+      if (!dst.acquire()) RETHROW(Error::LOST_RACE);
```

Comment:
This is different from the THROW on the failed acuire in MemMapper::createPageMap. Any reason?

Reply:
In my opinion, `MemMapper` is wrong. `acquire` already uses `THROW` on error, callers should `RETHROW` it. We don't actually have formulated a policy about this, but this is what would make sense to me.

"Fix" in next commits.